esmf_regrid/tests/unit/schemes/test_regrid_rectilinear_to_rectilinear.py

```diff
@@ -176,6 +176,12 @@ def test_laziness():
     assert result.has_lazy_data()
     assert np.allclose(result.data, src_data)
 
+    src.transpose()
+    assert src.has_lazy_data()
+    result = regrid_rectilinear_to_rectilinear(src, tgt)
+    assert result.has_lazy_data()
+    assert np.allclose(result.data, src_data.T)
```

Comment:
Thats correct, while I was looking through the code for `_map_complete_blocks` I realised that if I was sorting the dimensions during the code it would be important to make extra sure all cases of dimension ordering would be tested.

Reply:
see https://github.com/SciTools-incubator/iris-esmf-regrid/pull/198/commits/bd512ea29631a58b73c6699fc0c86c00710d1357